codestodians/food-basket-front-end
# [Feat: #150368365]  Add and Redesign Login/Signup Pages

#### What does this PR do?
-  Adds a dedicated page for `Login/Signup`
-  Updates the `Box` Component

#### Description of Task to be completed?
- Redesign Login/Signup Modal and contents(including text fields and button)

#### How should this be manually tested?
- Click on the `Login or Signup` Button on the top right to see the modal rendered for login/signup
- Navigate to `http://localhost:8080/login` to see the dedicated page (not modal) for login/signup

#### What are the relevant pivotal tracker stories?
[#150368365](https://www.pivotaltracker.com/story/show/150368365)

#### Screenshots (if appropriate)
<img width="420" alt="screen shot 2017-12-22 at 3 57 42 pm" src="https://user-images.githubusercontent.com/22661019/34302388-e3735b06-e730-11e7-873d-9209299a0b24.png">
<img width="451" alt="screen shot 2017-12-22 at 3 57 29 pm" src="https://user-images.githubusercontent.com/22661019/34302402-eb4cde06-e730-11e7-8fb4-575a2c001fb3.png">
